src/vs/workbench/services/editor/common/editorResolverService.ts

```diff
@@ -59,6 +59,21 @@ export interface IEditorType {
 configurationRegistry.registerConfiguration(editorAssociationsConfigurationNode);
 //#endregion
 
+//#region defaultBinaryEditor
+export const defaultBinaryEditorSettingId = 'workbench.defaultBinaryEditor';
```

Comment:
I am fine if we move this setting to the place where other `workbench.editor.xy` settings are defined.

Reply:
We define `editorAssociations` here and this is like a subset of it. Do you still want the setting moved?

Reply:
Moved to where the group lock setting is to allow for intellisense. I also will want to move editorAssociations some point to work on intellisense as well. Is it ok to move there?

Reply:
Feel free to move `editorAssociations` to the place where editor settings are contributed.